frontend/discourse/app/services/composer.js

```diff
@@ -1317,8 +1318,6 @@ export default class ComposerService extends Service {
       this.selectedTranslationLocale = null;
     } catch (e) {
       popupAjaxError(e);
-    } finally {
-      this.set("model.loading", false);
```

Comment:
`finally` gets called after regardless of try/catch, but `model` no longer exists after `.close()`. We don't need this here, we need it before we `close()`. Loading doesn't interfere with `popupAjaxError()` so we're covered there.